.travis.yml

```diff
@@ -1,16 +1,44 @@
 language: python
 
-python:
-  - 2.7
-  - 3.5
-
-env:
-  - TOXENV=django18
-  - TOXENV=django19
-  - TOXENV=django110
-  - TOXENV=coverage
-  - TOXENV=postgresql
-  - TOXENV=flake8
+matrix:
+  include:
```

Comment:
Why not using `exclude`? That will probably require less configuration, and highlight incompatible combinaisons.

Reply:
I preferred the "explicit" way, less sensitive to changes, and it allows to map the tox targets instead of defining specific ones (per django version) for travis

Reply:
Fair enough!